KarhooUISDK/UIComponents/LoyaltyView/KarhooLoyaltyPresenter.swift

```diff
@@ -16,7 +16,8 @@ final class KarhooLoyaltyPresenter: LoyaltyPresenter {
     }
     
     weak var delegate: LoyaltyViewDelegate?
-    private weak var view: LoyaltyView?
+    weak var internalDelegate: LoyaltyPresenterDelegate?
```

Comment:
It'd be better to rename `delegate` with, for instance, `viewDelegate` and use `delegate` identifier for `LoyaltyPresenterDelegate` type instance

Reply:
The `LoyaltyViewDelegate` will remain `delegate` and the `internalDelegate` has been renamed to `presenterDelegate` on both iOS and Android in the error handling ticket